youtube_dl/extractor/generic.py

```diff
@@ -2874,12 +2874,17 @@ def merge_dicts(dict1, dict2):
         # Look for HTML5 media
         entries = self._parse_html5_media_entries(url, webpage, video_id, m3u8_id='hls')
         if entries:
-            for entry in entries:
-                entry.update({
+            if len(entries) == 1:
+                entries[0].update({
                     'id': video_id,
-                    'title': video_title,
-                })
-                self._sort_formats(entry['formats'])
+                    'title': video_itle})
```

Comment:
`F821 undefined name 'video_itle'`.

Reply:
have no idea how that happend... must have hit x before :wq or something stupid... thx